app/controllers/alchemy/pages_controller.rb

```diff
@@ -4,6 +4,9 @@ class PagesController < Alchemy::BaseController
     before_action :enforce_primary_host_for_site
     before_action :render_page_or_redirect, only: [:show]
 
+    before_action :run_on_page_layout_callbacks, only: :show,
+      if: -> { OnPageLayout.callbacks.present? }
```

Comment:
Style/AlignHash: Align the elements of a hash literal if they span more than one line.